### DISCUSSION TITLE
[Request]: New Models

### DISCUSSION THREAD

**Discussion by AtomikkuSan** [zer0Kerbal/Dreamer#77] [General]
### Description of the new feature / enhancement

New Models!

Hey! Can we ask Randazzo if he has the time to redo the models?

### Proposed technical implementation details

New models for all parts.

[a comment by an automated account is not shown]

**Comment by FuryFire2004** [comment]
I'm currently working on new models here and there but please ask him

**Comment by FuryFire2004** [comment]
I suck at it

**Comment by FuryFire2004** [comment]
@AtomikkuSan like I beg of you please if you have someone to take over texturing from me do it ;-;(I don't think I know what I'm even doing lol)

**Comment by zer0Kerbal** [comment]
Let's get this release done before trying to change the models / textures -
